tests/cancun/eip4844_blobs/test_blob_txs.py

```diff
@@ -1020,6 +1020,7 @@ def test_invalid_blob_tx_contract_creation(
                     TransactionException.TYPE_3_TX_CONTRACT_CREATION,
                 ],
                 header_verify=header_verify,
+                skip_exception_verification=True,
```

Comment:
why?

are you making an invalid rlp type of test? 
this is a special group of tests. we must think about it

Reply:
Could you add a comment to explain why? :pray:

Reply:
This is actually what we are already doing for EELS filled tests: all exceptions in the mapper there are variations of "Invalid transaction", trimmed to different lengths, so it's ineffective at the moment.

I think we should open a issue to remove this, and another one to EELS to add strings to each transaction exception, it should not be that many.